src/dashboard/Push/PushDetails.react.js

```diff
@@ -39,7 +39,13 @@ const EXP_STATS_URL = 'http://docs.parseplatform.org/ios/guide/#push-experiments
 let getMessage = (payload) => {
   if(payload) {
     let payloadJSON = JSON.parse(payload);
-    return payloadJSON.alert ? payloadJSON.alert : payload;
+		if (payloadJSON.alert.body) {
```

Comment:
That would be ideal! Do you think you can do it? I think it makes sense to do it the best way we can if you are already touching these files.

And did you see my question? Don't we need to check the type of `payloadJSON.alert`, or it will always be an object here?